module/actor.js

```diff
@@ -93,16 +151,27 @@ class DCCActor extends Actor {
     const speaker = { alias: this.name, _id: this._id }
     const formula = `1d20 + ${weapon.toHit}`
 
+    /* Determine attack bonus */
+    const attackBonusExpression = this.data.data.details.attackBonus || null
+    let attackBonus = 0
+    if (attackBonusExpression) {
+      const abRoll = new Roll(attackBonusExpression)
```

Comment:
This allows putting a dice expression in the attack bonus box. Eg. +1d3 for a level 1 warrior.
Then by adding +@ab to the toHit and damage fields (eg. "+1+@ab") the variable attack bonus gets rolled once and applied to attack and damage for deed and deed-like abilities.
The roll output doesn't really show this well yet and if we had permission to implement more specific rules I'd probably just add a deed checkbox.